docs/readmes/orc8r/dev_dependencies.md

```diff
@@ -0,0 +1,13 @@
+---
+id: dev_dependencies
+title: Module Dependencies
+hide_title: true
+---
+
+# Module dependencies on generated constructs
+
+Within Magma, each module(orc8r, lte, cwf, feg, wifi, fbinternal),
+is in charge of generating and defining their own constructs. A module's
+constructs may have dependencies on those of another module.
```

Comment:
"A module's Swagger objects may depend on those from other modules. This results in the following module dependency tree"